packages/nimble-components/src/table/specs/table-column-sort-hld.md

```diff
@@ -34,15 +25,24 @@ The column definition will also be responsible for specifying the appropriate so
 
 For performance reasons, the table will not support custom sort functions on a column. Additionally, a column can only be sorted based on a single field within the table's data. These requirements guard against performance degradations from inefficient custom sort functions.
 
+Interactive sorting will be controlled by a new `sortMode` property on the `Table` (of type `TableInteractiveSortMode`):
+
+-   None (`undefined`, default): Interactive sorting is disabled
```

Comment:
I went back and forth on this a few times. I figured that we could enable interactive sorting by default at the SystemLink `sl-tbd` level if we wanted to. And having Multiple=`undefined` as the default (with `'single'` and `'none'` being the other values) felt a little weird (just how the enum values would end up looking). But neither of those are particularly strong justifications.

Reply:
What is driving the need to a sort mode property on the table at all? Do we have concrete use-cases of when a client would want to disable sorting on the table (but not disable it on each column) or when a client would want to restrict a user to only sorting at one column at a time?

Reply:
I'm not aware of any use cases that require the sort mode property (I was probably just presuming more configurability). Since we don't have a use case for it right now (and the existing `sl-grid` always has sorting on currently) we can omit `sortMode` until/unless we need it.

Updated the HLD to remove `sortMode`.